src/webgpu/web_platform/copyToTexture/canvas.spec.ts

```diff
@@ -28,6 +32,53 @@ function formatForExpectedPixels(format: RegularTextureFormat): RegularTextureFo
 }
 
 class F extends CopyToTextureUtils {
+  private anotherProvider: DeviceProvider | undefined;
+  /** Must not be replaced once acquired. */
+  private anotherDevice: GPUDevice | undefined;
+
+  getAnotherDevice(): GPUDevice {
+    assert(this.anotherProvider !== undefined, 'No another provider available right now;');
+    if (!this.anotherDevice) {
+      this.anotherDevice = this.anotherProvider.acquire();
+    }
+    return this.anotherDevice;
+  }
+
+  protected async init(): Promise<void> {
+    await super.init();
+
+    // Create another device which is not the default one.
+    this.anotherProvider = await devicePool.reserve({
+      requiredFeatures: ['texture-compression-bc'],
```

Comment:
(note: I think selectMismatchedDeviceOrSkipTestCase gets around this problem by using a separate DevicePool.)